src/core/behavior/shortcut.js

```diff
@@ -0,0 +1,63 @@
+import {ShortcutManager} from './shortcut.manager';
+
+const shortcutManager = new ShortcutManager();
+const codeMap = new Map()
+	.set(8, 'backspace')
+	.set(9, 'tab')
+	.set(13, 'enter')
+	.set(27, 'escape')
+	.set(32, 'space')
+	.set(33, 'pageUp')
+	.set(34, 'pageDown')
+	.set(35, 'end')
+	.set(36, 'home')
+	.set(37, 'left')
+	.set(38, 'up')
+	.set(39, 'right')
+	.set(40, 'down')
+	.set(113, 'f2')
+	.set(118, 'f7');
+
+
+export class Shortcut {
+	constructor(manager) {
+		this.manager = manager;
+	}
+	get keyCode() {
+		return Shortcut.keyCode;
+	}
+
+	get isControl() {
+		return Array.from(codeMap.values()).some(val => val === Shortcut.keyCode);
```

Comment:
can u add here another control values like ctrl F1 etc.?
and please add argument isControl(code),
Thanks!